alpha/apps/kaltura/modules/extwidget/actions/thumbnailAction.class.php

```diff
@@ -458,6 +461,10 @@ public function execute()
 		{
```

Comment:
Check of partnerCacheAge is configured and handle it before the case of _NOCACHE_